ModernWallet/Classes/Extension/UIView+Extensions.swift

```diff
@@ -18,4 +18,32 @@ extension UIView {
         viewLayer.shadowOffset = offset
     }
     
+    func pinToSuperview() {
+        self.translatesAutoresizingMaskIntoConstraints = false
+        
+        let views = ["pinnedView": self]
+        let horizontal = NSLayoutConstraint.constraints(withVisualFormat: "|-0-[pinnedView]-0-|",
```

Comment:
To stick the **pinnedView** to the leading and trailing edges you don't need to specify ```-0-``` distance to the edge, just change it to ```|[pinnedView]|```.